compiler/src/dmd/dtoh.d

```diff
@@ -2921,8 +2921,18 @@ public:
         auto check = cast(void*) (td ? td : sym);
 
         // Omit redundant fwd-declaration if we already emitted the entire declaration
-        if (visited.get(check, false))
-            return;
+        version (none)
+        {
+            // FIXME: use this version when 'get' can infer nothrow
+            if (visited.get(check, false))
+                return;
+        }
+        else
+        {
+            if (const pVisited = check in visited)
+                if (*pVisited)
+                    return;
+        }
```

Comment:
Note, this code should be compilable by compilers as old as 2.076. If `nothrow` isn't inferred, then you'll have to make it explicit.

Reply:
[Per run.dlang.io the following code doesn't compile with any version of DMD:](https://run.dlang.io/is/5DfcGK)
```d
int main() nothrow
{
    int[int] aa;
    return aa.get(1, 0);
}
```